protos/evm.proto

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 Intel Corporation
```

Comment:
I thought the only time we said that was really true was in the examples directory.